lib/helpers.js

```diff
@@ -358,7 +358,7 @@ function buildStartCmd (startAppOptions, apiLevel) {
     stopApp,
     flags,
   } = startAppOptions;
-  const cmd = ['am', 'start'];
+  const cmd = ['am', (apiLevel < 26) ? 'start' : 'start-activity'];
```

Comment:
we should also reuse this API call in the activateApp method to avoid unnecessary duplication

Reply:
Yea, they were not simple replacement so it will be another pr